src/store/index.ts

```diff
@@ -258,6 +263,15 @@ async function alertMatchedErrorAction(this: Store<any>, {
 
 /* --- MUTATIONS --- */
 
+function completeCodeWordMutation(state: any): void {
+    const codeParts = state.code.split(CODE_DELIMITER);
+    // Replace last (incomplete) word `codeWord`
+    codeParts.splice(codeParts.length - 1, 1, state.suggestedWord);
```

Comment:
Would this code always replace the input code with `suggestedWord` irrespective of the user input?